packages/eslint-config-ckeditor5/eslint.config.mjs

```diff
@@ -285,6 +285,13 @@ const rulesTypeScript = [
 			ts
 		},
 
+		languageOptions: {
+			parserOptions: {
+				projectService: true,
+				tsconfigRootDir: import.meta.dirname
```

Comment:
Isn't `import.meta.dirname` points to `/.../packages/eslint-config-ckeditor5`? There is no `tsconfig.json` or similar file. How is it supposed to work?